### ISSUE TITLE
Error formatting document when namespace includes keyword

### ISSUE THREAD

**Issue by Vac1911** [prettier/plugin-php#1930]
**@prettier/plugin-php v0.18.1**
[Playground link](https://loilo.github.io/prettier-php-playground/#N4IgDgTgpgLjCWUIHkwIPYDsDOIBcok8mMA6vACYwAW+AHAAwA0IMAhgEblW14AsLAK7YoAFU648AMzYAbES2zEA5rKgBFQehhR8M+VBZhqYAGpIlWfCDoA6BiBYwIbeLJUBhdAFtvbAAoAEv74zoKGIBwuAMZQAMowAJ5q1mDYEAC0AEyOINAAjoLw0P4uyn56cgogxCIQMKVs5WyVBgC+LFAU8DDoKGjwWJKgbNgwrSIdNZhgguN4IAA8APzGYAA6mJuYbN5Q2GBssQAEAIJgG+sAoiQ9iQDcm5vCUGcX69e3SR8AkiRIMliHxuCCSfx0EEBUEeW0wAHo4ccALJ9V5jQRSKTHahIXRtIA)

**Input:**
```php
<?php

namespace App\Entity;

use App\Entity\Interface\EntityInterface;

// More stuff here
```

**Output:**
```php
Parse Error : syntax error, unexpected 'Interface' (T_INTERFACE), expecting ';' on line 5

  3 | namespace App\Entity;
  4 |
> 5 | use App\Entity\Interface\EntityInterface;
    |               ^
  6 |
  7 | // More stuff here
```

**Expected behavior:**
When a namespace includes a keyword, it should not error, as these are technically valid. Note that renaming the namespace to  `Interfaces` does not error.

**Comment by czosel** [comment]
Hi @Vac1911, thanks for the bug report! This is a problem in our parser, potentially belonging to the same class as https://github.com/glayzzle/php-parser/issues/834.

**Comment by czosel** [comment]
Fixed in `v0.18.5` :tada:
